engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/VectorBuilderTest.java

```diff
@@ -0,0 +1,209 @@
+package org.enso.interpreter.test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.net.URI;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeHelpers;
+import org.enso.interpreter.runtime.warning.Warning;
+import org.enso.interpreter.runtime.warning.WarningsLibrary;
+import org.enso.interpreter.runtime.warning.WithWarnings;
+import org.enso.test.utils.ContextUtils;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.Value;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+public class VectorBuilderTest {
+  @ClassRule public static final ContextUtils ctx = ContextUtils.createDefault();
+  private static Value create;
+
+  @BeforeClass
+  public static void initCreate() throws Exception {
+    final URI srcUri = new URI("memory://longs.enso");
+    final Source src =
+        Source.newBuilder(
+                "enso",
+                """
+    import Standard.Base.Data.Vector.Builder
```

Comment:
These JUnit tests with small Enso code snippets are easier to debug than regular `test/*_Tests`. This is especially true when one deals with core Enso types like `Vector` & co. Because the `Standard.Test` harness is using these core Enso types as well it is really hard to debug them in _isolation_. Any breakpoint in `Vector` builtin is hit many times because `Standard.Test` uses the same code as the code under the tests. This is not happening with JUnit tests - the harness is fully in Java and the only `Vector` builtin calls are really made by the Enso snippet being tested.